bikeshed/boilerplate.py

```diff
@@ -363,7 +363,7 @@ def disambiguator(ref, types, specs):
     if ref.for_:
         try:
             disambInfo.append("for {}".format(", ".join(x.strip() for x in ref.for_)))
-        except:
+        except ValueError:
```

Comment:
Not sure about this one

Reply:
I have no clue what originally triggered me to add this. Regardless, it doesn't happen anymore; I've just pushed a commit to remove this try/except.